.changeset/config.json

```diff
@@ -9,5 +9,4 @@
   "access": "public",
   "baseBranch": "master",
   "updateInternalDependencies": "patch",
-  "ignore": ["fixture-direct", "fixture-simple", "fixture-server", "fixture-phrase"]
```

Comment:
It might work as expected [soon](https://github.com/changesets/changesets/discussions/783) :tm:. Really just trying to appease the bot, I'll add it back with `@fixtures/*`.

Reply:
👌 

![wait](http://i.imgur.com/8PfsQ.png)

Reply:
> just trying to appease the bot

![free your mind](https://c.tenor.com/sItb6CS4pYwAAAAM/morpheus-free-your-mind.gif)